Examples/Tests/point_of_contact_EB/analysis.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+import numpy as np
+from openpmd_viewer import OpenPMDTimeSeries
+import yt
+
+yt.funcs.mylog.setLevel(0)
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+# Open plotfile specified in command line
+filename = sys.argv[1]
+test_name = os.path.split(os.getcwd())[1]
+checksumAPI.evaluate_checksum(test_name, filename, output_format='openpmd')
+
+ts_scraping = OpenPMDTimeSeries('./diags/diag2/particles_at_eb/')
+
+
+it=ts_scraping.iterations
+x,y,z=ts_scraping.get_particle( ['x','y','z'], species='electron', iteration=it )
+
```

Comment:
Could you add a comment to explain that these are the analytically-derived coordinates of the point of the contact between the electron and the embedded boundary (given the electron initial position and velocity).